fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalDataSource.java

```diff
@@ -37,14 +37,15 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
  * External data source for elasticsearch
  */
 @Getter
 public class EsExternalDataSource extends ExternalDataSource {
 
-    public static final String DEFAULT_DB = "default";
+    public static final String DEFAULT_DB = "default_es_db";
```

Comment:
can we use the  `cluster_name`_`uuid` for DB?

Reply:
A catalog can only support one cluster, so it would be easier to use a static name.